Covarpop (#5340)
Codecademy/docs: content/mysql/concepts/build-in-functions/terms/covarPop/covarPop.md

---
Title: 'MySQL: COVAR_POP() function' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
Description: 'Covariance built-in function comparing the population of two cells in a table.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)

> Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
>   - 'Built-In Function'
>   - 'MySQL'

Reviewer: ```suggestion
Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
  - 'Built-In Function'
  - 'MySQL'
```
Include only those subjects that are part of the subjects.md list, if you feel that the list is insufficient then you can add a new subject in the [subjects.md file.](documentation/subjects.md)
Also, remove the comment ' # Please only use Subjects in the subjects.md file, for  more readability.

Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
  - 'Comparison'
  - 'Database'
  - 'MySQL'

> CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
>   - 'learn-MySQL-course'

Reviewer: ```suggestion
CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
  - 'learn-MySQL-course'
```
Include only those course/paths that are relevant to the particular concept and are available in the [catalog-content.md](documentation/catalog-content.md) file.

---

> The **COVAR_POP** function is the way we get a full covariance in MySQL as opposed to the COVAR_SAMP function. This takes two variables
> and returns the covariance in a decimal format. 

Reviewer: ```suggestion
The **`COVAR_POP`** function in MYSQL calculates the full covariance between two variables, which returns the result in decimal format. This function is different from `COVAR_SAMP`, which calculates the sample covariance.

```
